src/sonic-frr/patch/series

```diff
@@ -1,6 +1,4 @@
 0001-Add-support-of-bgp-tcp-DSCP-value.patch
 0002-Reduce-severity-of-Vty-connected-from-message.patch
-0003-Use-vrf_id-for-vrf-not-tabled_id.patch
```

Comment:
This patch is required. Otherwise fpmsyncd will break at https://github.com/Azure/sonic-swss/blob/master/fpmsyncd/routesync.cpp#L46

Reply:
@prsunny Yes. I understand. I need to remove it, to build an image and run the tests.